api/v1alpha1/package_test.go

```diff
@@ -46,6 +46,6 @@ notactuallyyaml
     `
 	ao := givenPackage(provided)
 	_, err := ao.GetValues()
-	assert.Error(t, err)
-	assert.Contains(t, err.Error(), "error unmarshaling")
+	assert.EqualError(t, err, "error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go value of type map[string]interface {}")
+	assert.Contains(t, err.Error(), "error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go value of type map[string]interface {}")
```

Comment:
I don't think both of these are necessary. Were it me, I'd keep the EqualError on 49, and delete line 50, but it also doesn't hurt anything I guess. But they're pretty much literally doing the same thing.